packages/faustwp-cli/index.ts

```diff
@@ -38,30 +35,11 @@ const config = new Configstore(CONFIG_STORE_NAME);
   }
   dotenv.config();
   validateFaustEnvVars();
-  /**
-   * If there is no config (or a non-valid config), prompt the user for their
-   * permission to collect anonymous telemetry information and save their
-   * preferences on their machine.
-   */
-  if (
-    !config.all?.telemetry ||
-    config.all?.telemetry?.enabled === undefined ||
-    !config.all?.telemetry?.anonymousId
-  ) {
-    /**
-     * Do not prompt for telemetry if preferences are not set and the command
-     * that is being ran is build or start. We do not want to halt the build of a
-     * production site that likely does not have preferences saved.
-     */
-    if (arg1 !== 'build' && arg1 !== 'start' && !disableCliInteraction()) {
```

Comment:
If prefs are not saved, we may want to `console.log` a statement telling them that telemetry exists. I had a WIP PR doing something like this. Maybe we can incorporate that message here?

https://github.com/wpengine/faustjs/pull/1176

Reply:
Added! Nice work with #1176!